.travis.yml

```diff
@@ -41,15 +38,18 @@ jobs:
           stage: Test
           after_success: beaver dlang codecov
           script:
+              - set -e
```

Comment:
Is all the items executed in the same shell? Otherwise the `set -e` here would be pointless. Also, travis checks the exit status of each individual command, so the `set -e` seems redundant in any case...

Reply:
Can't remember why I added `set -e` but it is definitely same shell.